src/features/common/components/CallToActionButton/CallToActionButton.js

```diff
@@ -0,0 +1,31 @@
+import PropTypes from 'prop-types';
+import { useHistory } from 'react-router-dom';
+
+import { CREATE_DONATION_PATH, CREATE_PETITION_PATH } from 'utils/constants';
+
+import styles from './CallToActionButton.module.scss';
+
+const CallToActionButton = ({ type }) => {
+  const history = useHistory();
+
+  const redirection = () =>
+    type === 'donation'
+      ? history.push(CREATE_DONATION_PATH)
+      : history.push(CREATE_PETITION_PATH);
+
+  return (
+    <button
+      className={`${styles[type]} ${styles.button}`}
+      type="button"
+      onClick={redirection}
+    >
+      <span>Crear {type === 'donation' ? 'donación' : 'solicitud'}</span> +
```

Comment:
Could you change this in order to get the onClick callback as a prop and the content of the button as a child?

The syntax could be something like this 

```
<CallToAcctionButton onClick={onCTAClick}>
  Crear donacion
</CallToActionButton>
```